.gitignore

```diff
@@ -33,11 +34,9 @@ language_voj.json
 /static
 backup*.json
 custom_script_*
-websocket/config.js
 node_modules/
-package-lock.json
 db.json
 tmp.sh
 *.sql
 fc_*
-local_urls.py
+local_urls.py
```

Comment:
Please add back the blank line

Reply:
@hieplpvip I tried in https://github.com/VNOI-Admin/OJ/pull/324/commits/3d7f306a319001e87325907450907bdb2667b628, but not sure if it's working.